loops_lab.js

```diff
@@ -0,0 +1,288 @@
+//
+//   ```
+//   2 bottles of beer on the wall, 2 bottles of beer.
+//   Take one down, pass it around, 1 bottle of beer on the wall.
+//   ```
+//
+//  In the last line, the word bottles (plural), is  replaced with bottle
+// (singular)
+//
+//   When the number of bottles is 1, the verse is:
+//
+//   ```
+//   1 bottle of beer on the wall, 1 bottle of beer.
+//   Take one down, pass it around, No more bottle of beer on the wall.
+//   ```
+//------------------MY answer
+let bottle = 99;
+while (bottle <= 99 && bottle >= 0) {
+  if (bottle >= 3) {
+    console.log(bottle + ' bottles of beer on the wall, ' + bottle + ' bottles of beer. Take one down, pass it around, ' + (bottle - 1) + ' bottles of beer on the wall.');
+  } else if (bottle === 2) {
+    console.log(bottle + ' bottles of beer on the wall, ' + bottle + ' bottles of beer. Take one down, pass it around, ' + (bottle - 1) + ' bottle of beer on the wall.');
+  } else if (bottle === 1) {
+    console.log(bottle + ' bottle of beer on the wall, ' + bottle + ' bottle of beer. Take one down, pass it around, No more bottles of beer on the wall.');
+  }
+
+  bottle--;
+}
+
+// ___
+//
+// 2. Use the `assignGrade` function (given below). Write a function that
+// logs each number from 60 - 100 along with its corresponding letter score.
+// Exp For each number from 81 to 90, log B, like so:
+//
+// ```js
+// 60 - F
+// ...
+// 81 - B
+// 82 - B
+// 83 - B
+// ...
+// 93 - A
+// 94 - A
+// ...
+// ```
+//
+// ```js
+// function assignGrade(score) {
+//     if (score > 90) {
+//         return 'A';
+//     } else if (score > 80) {
+//         return 'B';
+//     } else if (score > 70) {
+//         return 'C';
+//     } else if (score > 65) {
+//         return 'D';
+//     } else {
+//         return 'F';
+//     };
+// };
+// ```
+// <details>
+// <summary>
+//   <b>Hint</b>
+// </summary>
+//   Explore this:
+//
+//   ```js
+//   let grade = assignGrade(80)
+//   console.log(grade)
+//   ```
+//   What is happening here? Can you explain it?
+// </details>
+//--------------------- My answer
+
+let xy = 60;
+function assignGrade(xy) {
+  if (xy > 90) {
+    return 'A';
+  } else if (xy > 80) {
+    return 'B';
+  } else if (xy > 70) {
+    return 'C';
+  } else if (xy > 65) {
+    return 'D';
+  } else {
+    return 'F';
+  };
+};
+
+while (xy >= 60 && xy <= 100) {
+  console.log(xy + ' - ' + assignGrade(xy));
+  xy += 1;
+}
+
+// ___
+//
+// 3. Given an integer N draw a square of N x N asterisks. Look at the
+// examples.
+//
+// Example 1:
+// Input: `let N = 2`
+//
+// Output:
+// ```js
+// **
+// **
+// ```
+//
+// Example 2:
+// Input: `let N = 3`
+//
+// Output:
+// ```js
+// ***
+// ***
+// ***
+// ```
+//
+// <details>
+//   <summary>
+//     Hint 1
+//   </summary>
+//   Try printing/logging a single line of * first.
+// </details>
+//
+// <details>
+//   <summary>
+//     Hint 2
+//   </summary>
+//   You will need 2 loops for this.
```

Comment:
Good demonstration of for and while loops. No major mistakes or feedback. Good job!